CandyMarket/CandyMarket/DataAccess/CandyRepo.cs

```diff
@@ -14,5 +17,45 @@ public CandyRepo(IConfiguration config)
         {
             connectionString = config.GetConnectionString("CandyMarket");
         }
+
+        public IEnumerable<Candy> GetByUserId(int userId)
+        {
+            var sql = @"
+                        select candies.CandyId, candies.CandyName, Candies.Manufacturer, candies.FlavorCategory
+                        from UserCandies
+	                        join Users
```

Comment:
I don't think join user is needed, since userCandies already contains userId

Reply:
No it doesn't, got a little overzealous in case we needed it, lol.